@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -338,15 +338,13 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
         const worldLines: number[][] = []; // lines in world space.
         const uvs: number[][] = [];
         const offsets: number[][] = [];
-        const projectedTileBounds = this.m_decodeInfo.targetProjection.projectBox(
-            this.m_decodeInfo.geoBox
-        );
+        const projectedBoundingBox = this.m_decodeInfo.projectedBoundingBox;
 
         let localLineSegments: number[][]; // lines in target tile space for special dashes.
 
-        const tileWidth = projectedTileBounds.max.x - projectedTileBounds.min.x;
-        const tileHeight = projectedTileBounds.max.y - projectedTileBounds.min.y;
-        const tileSizeInMeters = Math.max(tileWidth, tileHeight);
+        const tileWidth = projectedBoundingBox.extents.x * 2;
+        const tileHeight = projectedBoundingBox.extents.y * 2;
+        const tileSizeWorld = Math.max(tileWidth, tileHeight);
```

Comment:
super minor, you could save a "* 2" by just multiplying the max by 2.

Reply:
ah true, I was not sure if tileWidth and height were used somewhere else, but they aren't